cmd/promtool/main.go

```diff
@@ -133,7 +133,11 @@ func main() {
 	dumpPath := tsdbDumpCmd.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
 	dumpMinTime := tsdbDumpCmd.Flag("min-time", "Minimum timestamp to dump.").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
-
+	importCmd := tsdbCmd.Command("create-blocks-from", "[Experimental] Import samples from input and produce TSDB blocks. Please refer to the storage docs for more details.")
+	omImportCmd := importCmd.Command("openmetrics", "Import samples from OpenMetrics input and produce TSDB blocks. Please refer to the storage docs for more details.")
+	importFilePath := importCmd.Flag("input-file", "Imports samples from file instead of stdin.").String()
+	importDBPath := importCmd.Flag("output", "Output directory for generated blocks.").Default(".").String()
```

Comment:
What do you think about my [earlier suggestion](https://github.com/prometheus/prometheus/pull/8084#issuecomment-720660677) to make the input and output flag _arguments_? That's IMHO better in line with what the other commands of `promtool tsdb` do. The default of the output dir should be `defaultDBPath` (i.e. "data/", not ".").

Reply:
yes, that suggestion sounds good I meant to implement that. sorry, an oversight on my part. Have changed it. 👍